Packs/SymantecEmailSecurity/Integrations/SymantecEmailSecurity/SymantecEmailSecurity_description.md

```diff
@@ -0,0 +1,57 @@
+## Summary
+This integration uses two Symantec APIs to manage email security.
+The **Symantec Email Security API** handles general operations, such as managing IOCs, data feeds, and email queues, providing real-time monitoring and threat management.
+The **Symantec Quarantine API** focuses on handling quarantined emails and manage allow/block lists for isolated emails.
+For more details, see the vendor's documentation [here](https://techdocs.broadcom.com/us/en/symantec-security-software/email-security/email-security-cloud/1-0/related-documents.html).
+
+## Credentials, URLs and Commands Mapping:
+- ***Symantec Email Security API - Credentials***:
+  - ***Server URL - IOC***
+    - `symantec-email-security-ioc-list`
+    - `symantec-email-security-ioc-action`
+    - `symantec-email-security-ioc-renew`
+  - ***Server URL - Data Feeds***
+    - `symantec-email-security-data-list`
+  - ***Server URL - Email Queue***
+    - `symantec-email-security-email-queue-list`
+- ***Symantec Quarantine API - Quarantine Credentials***:
+  - ***Server URL - Quarantine***
+    - `symantec-email-security-quarantine-email-preview-list`
+    - `symantec-email-security-quarantine-email-release`
+    - `symantec-email-security-quarantine-email-delete`
+    - `symantec-email-security-item-allow-list-list`
+    - `symantec-email-security-item-allow-list-update`
+    - `symantec-email-security-item-allow-list-delete`
+    - `symantec-email-security-item-block-list-list`
+    - `symantec-email-security-item-block-list-update`
+    - `symantec-email-security-item-block-list-delete`
+
+## Test Connection
+After inputting the **Credentials** or **Quarantine Portal Credentials**, click **Test** to validate the connection.
+The test will pass if only one set of valid credentials has been entered and the other left empty, or both enetered and valid.
```

Comment:
no need

Reply:
Why not explain to users the conditions for a valid connection?

Reply:
we are not usually do this.